content/docs/manual/static-ips.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Static IPs
+description: 'The list of IPs Convoy cloud servers send webhooks from.'
+id: static-ips
+order: 3
+---
+
+# Convoy IPs
+
+Convoy cloud servers operate from a static IP address. The IPs listed below represent Convoy cloud:
```

Comment:
@Youngestdev 

We need to have two segments here: 

- Static IPs
  - Static IP addresses are how we ensure all webhook events originate from the correct source without malicious entities impersonating the source. This added security was added for users with strict firewall restrictions. If you're deploying your Convoy instance, please read our deployment guide on how to set the egress server to ensure IP addresses don't change. 

- Convoy Cloud IPs
  - For our cloud customers, who are subscribed on a plan with Static IPs, all webhook events will originate from the following IP addresses: